FlipperZero/.swiftlint.yml

```diff
@@ -5,11 +5,81 @@ excluded:
  - Frameworks
 
 disabled_rules:
- - force_try
  - identifier_name
- - line_length
- - force_cast
- - trailing_whitespace
- - opening_brace
+ - todo
```

Comment:
Maybe it's better to keep `todo` bc it would be better visible in code and easely distinguishable from other comments.
Also Xcode have no special place to track TODOs, so it is quite convenient to see them on issues pane. 
Alternatively if the rule would be disabled we can control TODOs visibility per case basis setting `#warning("TODO: something")` in critical places.

Reply:
One of the purposes for the linter integration is to verify there are no new issues in code changes. Because we are not going to fix all of the TODOs, we will see those warnings in every PR. So, we'll have to ignore them, which defeats the purpose of enabled rule.
I expect to enable it back on the later stage.

Reply:
It is legit point not to overwhelm issues with todo’s warnings.